### DISCUSSION TITLE
Not receiving XCH rewards

### DISCUSSION THREAD

**Discussion by REIS77-COMMITS** [Chia-Network/chia-blockchain#7837] [Support]
Hello all,

I am having a problem. Can anyone help me with it? I have this showing up in my wallet history: Incoming | xch16s6mvftf7w528u336rrggt8594594e7zmn3d58nz2nz6dej85yyswm86q4 | 07/10/2021 18:05:37 | Confirmed at height 552369 | 1,749999999999 but no such amount in my wallet. My full node and wallet are synced. I am using Chia Blockchain in two different computers with the same seed phrases. My second computer does not show that transaction in wallet history. I have already installed the latest version of Chia Blockchain in both computers.

Thanks,

**Comment by SlugFiller** [comment]
The fact that it's exactly 1 mojo less than an actual block reward, indicates to me that this is a change transaction. My guess is that this will be very close to a 1 mojo outgoing transaction, corresponding with creating a pool NFT.

This is a self-transaction. The input and output are both your own wallet, so no extra XCH would appear in your balance. It seems the wallet client has issues recognizing change transactions, probably due to the way transactions are built, so they appear in the history.

**Comment by REIS77-COMMITS** [comment]
Hi,

Thank you very much for the explanation.

Best,

28 Tem 2021 Çar 16:28 tarihinde SlugFiller ***@***.***> şunu
yazdı:

**Comment by Peppins32** [comment]
I have same issue:

PC1
![immagine](https://user-images.githubusercontent.com/74967219/127702141-4454d9fa-b5e6-45eb-ae4a-cca74762631e.png)

PC2
![immagine](https://user-images.githubusercontent.com/74967219/127702381-f791f771-c767-4475-a548-abd28fdc601c.png)

Need to do something? I copyed wallet/db from pc2 to pc1 and now on both i see same thing (outgoing).

**Reply by SlugFiller** [reply]
Again, you can see the outgoing mojo in PC2, and you can tell that the block heights are identical to the incoming change in PC1. I'm not entirely sure why the client can't tell the outgoing transaction from the incoming change. I have read somewhere that Chia's blockchain format doesn't have a concept of transactions, only of Chia going into and out of an address, with the requirement that the numbers balance out. So maybe that's why it has trouble telling a change address from an incoming transaction, since it can't tell it's coming from the same wallet, or that it's associated with the outgoing transaction.
